djredcap/management/subcommands/convert.py

```diff
@@ -39,11 +39,13 @@ class Command(BaseCommand):
     args = 'filename'
 
     json_filename = ''
+    #add a command line option -m allowing the user to set a filename for models
+    #filename is RELATIVE to the current working directory
+    BaseCommand.option_list = BaseCommand.option_list + (make_option('-m', '--modelfile', dest='modelfilename', help='filename to which models are written'),)

```

Comment:
Remove this option since it is defined below. On a side note, you are mutating the `BaseCommand.option_list` not the current command.